web/docs/how-does-tenzir-work/README.md

```diff
@@ -0,0 +1,66 @@
+# How Does Tenzir Work?
+
+Tenzir revolves around three primary components:
+
+- **Pipeline**: A defined sequence of operators responsible for loading,
+  parsing, transforming, and routing data. Pipelines are the core mechanism for
+  data processing.
+- **Node**: A running process that manages and executes pipelines.
+- **Platform**: A higher-level management layer that provides oversight and
+  control over multiple nodes.
+
+## Standalone and Managed Pipelines
+
+Tenzir offers flexibility in running pipelines through two main modes:
+
+1. **Standalone Execution**: Run directly from the command line using the
+   `tenzir` binary. This mode is ideal for quick, manual data transformations
+   akin to how one would use `jq`, but with broader data handling capabilities.
+2. **Managed Execution via Nodes**: For more sustained or automated data
+   processing, pipelines can be managed within a node. This is either launched
+   as a Docker container or by running the `tenzir-node` binary.
+
+![Pipelines Two Ways](user-journey-1.svg)
+
+The standalone option caters to interactive, ad-hoc tasks, while managed
+pipelines are better suited for continuous or scheduled dataflows. Notably,
+managed pipelines can handle both long-term and short-duration jobs.
```

Comment:
Do you want to mention extra features available to managed pipelines, such as stored metrics and stored diagnostics? Maybe not needed here, idk.